minard/templates/pca_tellie_fibre.html

```diff
@@ -0,0 +1,660 @@
+{% extends "layout.html" %}
+{% block title %}PCA Fibres{% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h1 style="text-align:center">TELLIE PCA Run: {{ run }}</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h3 class="panel-title"><b>Beamspot fit</b></h3>
+          </div>
+          <div class="panel-body">
+            {% for run in data %}
+            {% if loop.index is divisibleby 2 %}
```

Comment:
Are there supposed to be a bunch of empty plots here? I have one figure and then a bunch of "alt" strings where the code is drawing empty data? In other words, is this looping over different figures?

Reply:
This loads couchdb document and then loops over datasets - so yes, should have multiple figures. The number is dynamic with the # of processed datasets. 
(This should represent fibre evolution over time. It's data for the same fibre, across all available datasets.)

Perhaps you have not mounted the full directory with all the plots - I can see all of them.

Reply:
OK sounds good, I must just be missing the files.